Source/Python/domain/descriptors.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, *, name: str, **kwargs) -> None:
         if len(name) == 0:
             raise ValueError(f"{self.__class__.__name__} attribute 'name' cannot be empty")
         if not match(r"^[\w\-. ]*$", name):
-            raise ValueError(f"{self.__class__.__name__} attribute 'name' must contain only alphanumeric, underscore, dash, and space characters")
+            raise ValueError(f"{self.__class__.__name__} attribute 'name' must contain only alphanumeric, underscore, dash, and space "
+                             f"characters")
```

Comment:
The regex allows the `.` character, which should be allowed for a name. The message should be updated to state this.

Reply:
Fixed